server/models/views/river-and-sea-levels.js

```diff
@@ -42,7 +42,7 @@ class ViewModel {
     this.referer = referer
     this.rivers = rivers
     this.floodRiskUrl = floodRiskUrl
-    this.isMultilpleMatch = this.checkMultipleMatch(rivers, stations)
+    this.isMultilpleMatch = !!rivers?.length
```

Comment:
I'd argue that `rivers?.length > 0` would express intent more clearly.

I had to investigate if `!!` was an operator (turns out it just double negation) to understand the code!

Reply:
we use the !! in a few places in the codebase, but yeah i do prefer readability of code over getting it as small as possible, Im happy to change it